README.md

```diff
@@ -1268,6 +1269,76 @@ TYK_PMP_PUMPS_CSV_TYPE=csv
 TYK_PMP_PUMPS_CSV_META_CSVDIR=./
 ```
 
+## SQS Config
+
+#### Authentication & Prerequisite
+
+We must authenticate ourselves by providing credentials to AWS. This pump uses the official AWS GO SDK, so instructions on how to authenticate can be found on [their documentation here](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials).
+
+#### Config Fields
+
+`aws_queue_name` - Specifies the name of the AWS Simple Queue Service (SQS) queue where messages will be sent
+
+`aws_message_group_id` - Specifies the name of the AWS Simple Queue Service (SQS) queue where messages will be sent
+
+`aws_sqs_batch_limit` - Sets the maximum number of messages to include in a single batch when sending messages to the SQS queue
+
+`aws_message_id_deduplication_enabled` - Enables or disables the deduplication of messages based on unique message IDs to prevent unintended duplicates in the queu
+
+`aws_delay_seconds` - Configures the delay (in seconds) before messages sent to the SQS queue become available for processing.
+
+When you initialize a Timestream Pump, the SDK uses its default credential chain to find AWS credentials. This default credential chain looks for credentials in the following order:
```

Comment:
@mativm02 I believe we have a typo here - it should say When you initialize a SQS Pump